net/libproxy/Portfile

```diff
@@ -18,94 +18,53 @@ long_description    Libproxy exists to answer the question: Given \
                     a network resource, how do I reach it? It handles all \
                     the details, enabling you to get back to programming.
 
-checksums           rmd160  18680c62196d6b1765864ac8d8cfb0da19e96fb3 \
-                    sha256  712f5965b57dc329fd5700f906d9943dbdc379e547e31dc8b11c77332a652d52 \
-                    size    98063
+checksums           rmd160  35c1118e961666ca15726ac9ffa899f30cd7355d \
+                    sha256  aed5d3ce5cfbf394595d79c717b04b61a6956540366bcbd0a73e2ae77ccfedea \
+                    size    58147
 
-patchfiles-append   patch-bindings-perl-src-CMakeLists.txt.diff \
-                    patch-libproxy-cmake.diff \
-                    patch-libproxy-test-CMakeLists.txt.diff
-
-depends_build-append \
-                    path:bin/pkg-config:pkgconfig
-
-depends_lib-append  path:bin/vala:vala \
-                    path:lib/pkgconfig/glib-2.0.pc:glib2
+depends_lib-append  \
+                    port:icu \
+                    port:gsettings-desktop-schemas \
+                    port:gobject-introspection
 
 # https://trac.macports.org/ticket/63267
 compiler.cxx_standard 2011
 
-# webkit and mozjs pacrunners disabled by default
-# due to the following issues:
-#
-# webkit pacrunner circular dependency problem
-# webkit-gtk -> libsoup -> libproxy -> webkit-gtk (#26261)
-#
-# pac files support not available on Tiger (#24127)
-#
-# TODO: add variants that safely enable these features
-
 use_parallel_build no
```

Comment:
Since this is now using a completely different build system, it seems likely that whatever was preventing the parallel build from working before isn't anymore; can you try if it still builds after removing this?

Reply:
Fixed